lang/elixir/Makefile

```diff
@@ -0,0 +1,69 @@
+#
+# Copyright (C) 2009-2020 OpenWrt.org
+#
+# This is free software, licensed under the GPL License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=elixir
+PKG_VERSION:=1.10.2
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/elixir-lang/elixir/archive/v$(PKG_VERSION)
+PKG_HASH:=5adffcf4389aa82fcfbc84324ebbfa095fc657a0e15b2b055fc05184f96b2d50
+
+PKG_MAINTAINER:=Xiaobo Tian <peterwillcn@gmail.com>
+PKG_LICENSE:=GPL-2.0
```

Comment:
This is deprecated [SPDX License Identifier](https://spdx.org/licenses).

Reply:
Updated

Reply:
Still no. It needs -or-later or -only.